.husky/pre-commit

```diff
@@ -2,3 +2,4 @@
 . "$(dirname "$0")/_/husky.sh"
 
 npx lint-staged
+ggshield secret scan pre-commit
```

Comment:
Out of curiosity, could we source our env var `GITGUARDIAN_API_KEY` from here? that way we dont need to add anything to our global variable in our machines!

Reply:
Sure, I thought by installing globally, devs can use it across all repos and also the global CLI to scan. But I think .env also works

Will update the README and command